src/prefect/core/flow.py

```diff
@@ -568,30 +568,54 @@ def chain(self, *tasks: Task, validate: bool = None) -> List[Edge]:
             )
         return edges
 
-    def update(self, flow: "Flow", validate: bool = None) -> None:
+    def update(
+        self, flow: "Flow", parmerge: bool = None, validate: bool = None
```

Comment:
I think we should clarify this kwarg name to `merge_parameters` and default it to `False` for clarity.